frontend/src/services/graphQLDevice.ts

```diff
@@ -66,6 +66,30 @@ const DEVICE_SELECT = `
         email
       }
     }
+  }
+  events {
+    hasMore
+    total
+    items {
+      id
+      state
+      timestamp
+      type
+      action
+      actor {
+          email
+      }
+      services {
+          id
+          name
+      }
+      users {
+          email
+      }
+      ... on DeviceShareEvent {
+          scripting
+      }
+    }
```

Comment:
Looks like you are getting the logs in it's own model now, can we remove this from the device list query?